src/plugins/mergeCells/mergeCells.js

```diff
@@ -128,6 +128,11 @@ export class MergeCells extends BasePlugin {
     this.addHook('beforeDrawBorders', (...args) => this.onBeforeDrawAreaBorders(...args));
     this.addHook('afterDrawSelection', (...args) => this.onAfterDrawSelection(...args));
     this.addHook('beforeRemoveCellClassNames', (...args) => this.onBeforeRemoveCellClassNames(...args));
+    this.addHook('beforeUndoStackChange', (action, source) => {
+      if (source === 'MergeCells') {
```

Comment:
Could you add a test case for the _MergedCells_ plugin that would cover that change?

Reply:
Current tests for MergeCells plugin cover UndoRedo plugin and fail without callback. I've checked it.

Reply:
![Screenshot 2021-04-29 at 10 59 43](https://user-images.githubusercontent.com/141330/116526586-0a0ea180-a8da-11eb-9c1a-b1ca7d5b8f84.png)

Reply:
Great!